src/SFA.DAS.EmployerAccounts.Web/Orchestrators/SearchPensionRegulatorOrchestrator.cs

```diff
@@ -80,7 +78,8 @@ private PensionRegulatorDetailsViewModel ConvertToViewModel(Organisation organis
                 Name = organisation.Name,
                 Status = organisation.Status,
                 Address = organisation.Address.FormatAddress(),
-                ReferenceNumber = organisation.UniqueIdentity
+                ReferenceNumber = organisation.UniqueIdentity,
+                Type = OrganisationType.PensionsRegulator
```

Comment:
Shouldn't this have resulted in a change to a test?

Reply:
I have added a test to cover this. However I don't really think this property should be being set at this 'high' level. I think really this 'source' property should be set way down at the point it is retrived in the `IPensionRegulatorService` implementation.

I had a quick look at doing that, but I think the best way would be to hook into the Json deserialization and didn't have a quick answer for that :0) 

We can sort that out as part of this PR if we want though. 

(also at some point renaming `OrganisationType` to something like `SourceType` might be helpful.)

Reply:
I agree it shouldn't be here although disagree that it should come from the service.  I'd say it only needs to be set by whatever method is pushing it back to be saved.

Reply:
That would make sense, although I have a fear this will be used as a 'flag' to switch between options somewhere later; not sure, but would not be surprised.